ecl/eclcc/eclcc.cpp

```diff
@@ -1306,6 +1307,14 @@ void EclCC::processSingleQuery(EclCompileInstance & instance,
 
             updateWorkunitTimeStat(instance.wu, SSTcompilestage, "compile:parse", StTimeElapsed, NULL, parseTimeNs);
 
+            if (optExtraStats)
+            {
+                updateWorkunitTimeStat(instance.wu, SSTcompilestage, "compile:cache", StNumAttribsSimplified, NULL, parseCtx.numAttribsSimplified);
```

Comment:
Minor:  More encapsulated if this was a single call: parseCtx.addWorkunitStats().  Can leave as it is though.